src/models/yamlProjectConfiguration.ts

```diff
@@ -462,6 +472,12 @@ export class YamlProjectConfiguration {
             workspace,
             configurationFile.packageManager,
         );
+        if (ENVIRONMENT === "dev") {
+            // Used for a/b testing of genezio runtime.
+            // TODO: remove after a/b testing is complete.
+            yamlProjectConfiguration.writeToFile();
```

Comment:
This should be done only if we change the provider.

Reply:
I doubt `zod` gives a function to determine if the default option has been selected or not.